src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -154,18 +154,19 @@ public string WarningsNotAsErrors
 
         #region Tool Members
 
-        internal override BuildProtocolConstants.RequestLanguage Language
-            => BuildProtocolConstants.RequestLanguage.CSharpCompile;
+        internal override BuildProtocolConstants.RequestLanguage Language => BuildProtocolConstants.RequestLanguage.CSharpCompile;
```

Comment:
Please keep wrapped. The long lines make it difficult to read on smaller monitors.